okta/komand_okta/actions/get_factors/action.py

```diff
@@ -0,0 +1,49 @@
+import komand
+from .schema import GetFactorsInput, GetFactorsOutput, Input, Output, Component
+# Custom imports below
+import requests
+import json
+from komand.exceptions import PluginException
+
+
+class GetFactors(komand.Action):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name='get_factors',
+            description=Component.DESCRIPTION,
+            input=GetFactorsInput(),
+            output=GetFactorsOutput())
+
+    def run(self, params={}):
+        user_id = params.get(Input.USER_ID)
+        if not user_id:
+            raise PluginException(cause='user_id is required')
+
+        try:
+            okta_url = self.connection.okta_url
+            url = requests.compat.urljoin(okta_url, f'/api/v1/users/{user_id}/factors')
+            response = self.connection.session.get(url)
+            if response.status_code not in range(200, 299):
+                raise PluginException(
+                    cause="Received HTTP %d status code from Okta. Please verify your Okta server status and "
```

Comment:
Looks like %d is not being substituted, use an f-string perhaps and the var name